pages/components/homePage/shortAboutUs/ShortAboutUs.tsx

```diff
@@ -22,9 +22,10 @@ export const ShortAboutUs = () => {
         <NavLink href="contact-us/">
           <HdOutlinedButton text={data["contact-us-button"]}/>
         </NavLink>
-        <NavLink href="services/">
-          <HdTextButton text={data["view-services-button"]}/>
-        </NavLink>
+        <HdTextButton
+          text={data["view-services-button"]}
+          onClick={()=> {window.scrollTo({ top: 1750, behavior: "smooth" });}}
```

Comment:
```suggestion
          onClick={()=> {window.scrollTo({ top: DYNAMIC, behavior: "smooth" });}}
```

fixed 1750 will broke in other screens